reference-implementation/unittests/tensor.cpp

```diff
@@ -58,4 +60,35 @@ TEST(tensor, extract) {
   }
 }
 
+TEST(tensor, splat) {
```

Comment:
Code was changed in #262

Reply:
I was aware of that, but seems I was on an out-of-date branch. Thanks for pointing me to this.